docs/installation.rst

```diff
@@ -0,0 +1,24 @@
+Installation
+============
+
+Willow supports Python 2.7+ and 3.3+. It's a pure-python library with no hard
+dependencies so doesn't require a C compiler for a basic installation.
+
+Installation using ``pip``
+--------------------------
+
+.. code-block:: shell
+
+    pip install Willow==0.3b3
```

Comment:
Quick note, this should now be "Willow==0.3". Could [|release|](http://www.sphinx-doc.org/en/stable/config.html#confval-release) do the trick here?